docs/detections/rules-ui-monitor.asciidoc

```diff
@@ -15,6 +27,30 @@ TIP: To sort by any other column, switch on the *Technical preview* toggle above
 
 For detailed information on a rule, the alerts it generated, and associated errors, click on a rule's name in the *All rules* table. This also allows you to perform the same actions that are available on the <<rules-ui-management, **Rules** tab>>, such as modifying or deleting rules, activating or deactivating rules, exporting or importing rules, and duplicating prebuilt rules.
 
+[float]
+[[rule-execution-logs]]
+=== Rule execution logs
+
+Each detection rule keeps a log of every time it ran, its success or failure, any warning or error messages, and how long it took to search for data, create alerts, and complete its execution. This can help you troubleshoot a particular rule if it isn't behaving as expected (for example, if it isn't creating alerts or takes a long time to run).
+
+To access a rule's execution log, go to *Detect* -> *Rules*, click the rule's name to open the rule details, then scroll down and select the *Rule execution logs* tab.
+
+[role="screenshot"]
+image::images/rule-execution-logs.png[Rule execution logs table]
+
+You can hover over each column heading to display a tooltip about that column's data. Click a column heading to sort the table by that column.
+
+Use these controls to narrow down what's included in the logs table:
+
+* The *Status* drop-down filters the table by rule execution status: 
```

Comment:
@spong Please take a close look at these statuses to make sure I described them correctly. Thanks!